src/dashboard/trulens/dashboard/run.py

```diff
@@ -100,9 +99,9 @@ def run_dashboard(
     # run leaderboard with subprocess
     leaderboard_path = static_resource("dashboard", "Leaderboard.py")
 
-    if Tru._dashboard_proc is not None:
-        print("Dashboard already running at path:", Tru._dashboard_urls)
-        return Tru._dashboard_proc
+    if session._dashboard_proc is not None:
```

Comment:
Modified from class._object to object._object is that alright?

Reply:
Not right now since object._object is not defined. I'm making Tru (now TruSession) a pydantic model which it should have been all along. With that change object._object will be correct.

Reply:
When _object is not an instance attribute, `object._object` will point to `class._object`. So this shouldn't functionally change anything right now but if/when TruSession is no longer a singleton we'd want to make class attributes like _dashboard_proc an instance attribute.